test.js

```diff
@@ -88,6 +88,16 @@ test('static node package', t => babelTest(t,
 	'import mod from "/node_modules/@cfware/fake-module1/index.js";'
 ));
 
+test('static package from a different resolve directory', t => babelTest(t,
```

Comment:
Could you add two more test?
* One just like this test except add the option `filename: 'fixtures/my-modules/my-other-module/foo.js'` to test one module in `fixtures/my-modules` importing another.
* Another test to simulate a module in `fixtures/my-modules` importing something from node_modules.

Reply:
Added the tests.